lib/class-wp-theme-json-gutenberg.php

```diff
@@ -724,7 +734,6 @@ private function get_block_classes( $style_nodes ) {
 			}
 
 			if ( self::ROOT_BLOCK_SELECTOR === $selector ) {
```

Comment:
Makes me wonder if all the content of this condition should be above but it's good regardless.